ghcup-prereleases-0.0.8.yaml

```diff
@@ -2141,13 +2141,165 @@ ghcupDownloads:
         dlSubdir: ghc-9.12.20241014
         dlUri: https://downloads.haskell.org/~ghc/9.12.20241014/ghc-9.12.20241014-src.tar.xz
       viTags:
-      - LatestPrerelease
```

Comment:
When removing `LatestPrerelease` you have to add `Prerelease` tag.

Reply:
done